src/bench.cpp

```diff
@@ -0,0 +1,86 @@
+#include "bench.hpp"
+#include "search.h"
+#include "search.hpp"
+#include "uci.hpp"
+#include "util/types.hpp"
+#include <array>
+#include <iostream>
+#include <string>
+
+namespace Clockwork {
+namespace Bench {
+const std::array<std::string, 50> BENCH_FENS = {
```

Comment:
Minor: maybe: (would also need to change the `for` loop)
```suggestion
const std::array<std::string_view, 50> BENCH_FENS = {
```

Reply:
Tbf since there is implicit conversion i would prefer if this remains a string, unless there is any benefit to it being a string_view.